### DISCUSSION TITLE
sync all header color schemes with dashboard view header color scheme

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [home-assistant/frontend#11075] [Dashboard]
changing color schemes for our views can be done by using card-mod-themes and set the variables for
```
        app-header, app-toolbar {
          background: radial-gradient(var(--primary-color),var(--card-background-color));
          color: var(--text-primary-color);
        }
```

<img width="675" alt="Schermafbeelding 2022-01-03 om 02 02 01" src="https://user-images.githubusercontent.com/33354141/147893880-5edcc869-56cd-40aa-9d88-ac36bb481eed.png">

however, this doesnt work on the left side menu pages, which either use the primary color, like dev tools and energy. 

<img width="868" alt="Schermafbeelding 2022-01-03 om 02 01 48" src="https://user-images.githubusercontent.com/33354141/147893889-c20c1603-0d9e-48b0-806a-3280c406133b.png">

Configuration uses yet another scheme, and doesnt set primary color as header.

<img width="1037" alt="Schermafbeelding 2022-01-03 om 02 01 38" src="https://user-images.githubusercontent.com/33354141/147893893-7df6b7a5-af5f-4c86-a598-a1f7d753697c.png">

Fully aware card-mod-theme'ing is custom and not core at all, I would like to ask if there is some purpose behind the big differences in color scheming between the 
- dashboard views, 
- the dev tools, and 
- the configuration

If core Lovelace could use the same variables/color scheme for these pages (preferably the scheme for dashboards), we could streamline the overall view of these pages in our themes.

thanks for considering

**Comment by mrand** [comment]
I would love to see this made more consistent, especially because in my mind, it's actually a bit worse than you describe:
 Some theme variables are overloaded and/or used inconsistently.  Unless things have changed recently, here are two examples:

- text-primary-color:   used for some text, and also tab indicator for dashboard and dev panels
- mdc-theme-primary: used for the background on some buttons, but text on other buttons

The end result of this is that there are certain things that are not possible to properly theme as desired, especially if your trying to work with a limited palette (my night theme is monochrome red).
